modules/administration-guide/pages/configuring-storage-sizes.adoc

```diff
@@ -57,4 +57,11 @@ spec:
 <2> Specify a claim size on the next line or omit the next line to set the default claim size value. The specified claim size is only used when you select this storage strategy.
 <3> The claim size must be specified as a link:https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/[{kubernetes} resource quantity]. The available quantity units include: `Ei`, `Pi`, `Ti`, `Gi`,  `Mi` and `Ki`.
 
-pass:[<!-- vale RedHat.CaseSensitiveTerms = YES -->]
\ No newline at end of file
+pass:[<!-- vale RedHat.CaseSensitiveTerms = YES -->]
+
+[IMPORTANT]
+====
+Manually modifying a PVC on the cluster that was provisioned by {prod-short} is not officially supported and may result in unexpected consequences.
+
+If you want to resize a PVC that is in use by a workspace, you must restart the workspace must before the PVC change can occur.
```

Comment:
Small typo, must is repeated twice "you must restart the workspace must before"

Reply:
ooops, fixing immediately